### ISSUE TITLE
Crashes repeatedly when encountering directories with .sh

### ISSUE THREAD

**Issue by Yzrsah** [mads-hartmann/bash-language-server#102]
In my directory structure, there's some directories called i.e. "Now.sh"

```
Analyzing file:///foo/CloudDeploy/Now.sh
fs.js:121
    throw err;
    ^

Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:516:3)
```

`The Bash IDE server crashed 5 times in the last 3 minutes. The server will not be restarted.`

This should not happen and analyzer should not attempt to read directories as files.

**Comment by skovhus** [comment]
Thanks for reporting this. Let us know if you want to try to fix this.

**Comment by Yzrsah** [comment]
Sorry I have a couple hundred issues open right now, I will deal with this later

**Comment by Yzrsah** [comment]
Sorry I have a couple hundred issues open right now, I will look at this later

**Comment by skovhus** [comment]
Will keep this open until we release the next version (hopefully soon)

**Comment by Yzrsah** [comment]
@skovhus Thank you very much. I'm lucky enough to enable this extension now

**Comment by Yzrsah** [comment]
@skovhus Thank you very much

**Comment by skovhus** [comment]
Thank you for fixing this. This was released as version 1.5.6.

Upgrade by running:
`npm install -g bash-language-server`
